_posts/2021-07-12-gs_00002_terminal_basic.md

```diff
@@ -0,0 +1,421 @@
+and, as mentioned earlier, your home directory "~". There are two more things
+you should know about the "cd" command. First, if you use "cd -" you change to
+the last folder you visited. Second, if you execute "cd" without any command you
+will change to your home directory. Try the following examples:
+
+      $ cd                  # change to your home directory
+      $ cd ~                # change to your home directory
+      $ cd Desktop          # change to your Desktop
+      $ cd -                # change to the last folder (here: home directory)
+      $ cd ./Desktop        # same as "cd Desktop"
+      $ cd ../Desktop/..    # Can you guess? Verify with "pwd".
+
+A few notes to make things faster for you. If you press the arrow-up key you can
+reuse your previously used commands. With CTRL-R you can search in your command
+history. Just play around with it, but in the beginning, we recommend typing the
+commands yourself. Especially, because the commands _pwd_, _ls_, and _cd_ are
+faster to type than to search for in your history. If you are a lazy typer, you
+also can highlight a command, for example, in your web browser and paste it into
+the terminal by pressing your mouse wheel.
+
+## Play with files and folders
+
+Next, we want to show how to deal with files and folders. You can create a file
+using the "touch" command with the file name as a parameter:
+
+    $ touch my_first_file
+
+List the content of the current folder and you should see the newly created file
+(Can you already recall the correct command?):
+
+    $ ls
+
+To create a folder, you can use the "make directory" command:
+
+    $ mkdir new_folder
+
+Again, you can use the "list" command to see your new folder. Let's copy the
+previously created file into the new folder:
+
+    $ cp my_first_file new_folder
+
+From the example, you can see that the structure of the command is:
+
+    $ cp source destination
+
+So far, we only used the "list" command to show the content of the current
+folder, but you also can list the content of other folders. In this case, we
+want to check if the file was really copied to "new_folder":
+
+    $ ls new_folder
+
+Similar to copying a file, you can also move the file:
+
+    $ mv my_first_file new_folder
+
+Here is one thing you should note! There is already a file named "my_first_file"
+in "new_folder". What will happen now? In a graphical interface, you usually
+will be asked if you really want to replace the existing file. If you try the
+command above, you will notice that this is not the case here. The existing file
+will simply be overwritten without notice.
+
+<div class='note-normal' markdown="1">
+_Attention: Stupid usage of the terminal can lead to loss of important data_
+</div>
+
+Lastly, we want to remove the file. Since we are lazy programmers, we do not
+want to type the whole path ourselves. Let's use **autocompletion** by pressing
+the [tab] key:
```

Comment:
How is it now? I think it should be clear. I stilled used $ because I wrote before that '"$” [...] indicates that the command should be executed in a terminal'

Reply:
Yes, that looks better.